pregenerate_asm/Cargo.toml

```diff
@@ -10,3 +10,4 @@ path = "../build.rs"
 # Keep this in sync with `[build-dependencies]` in ../Cargo.toml.
 [dependencies]
 cc = "1.0.26"
+tempfile = "3.0"
```

Comment:
Please add a comment explaining why this dependency was added. If it is only needed for PPC then the dependency should be limited to PPC.